awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -214,108 +234,73 @@ function JobTemplateForm({
-              isValid={!form.touched.project || !form.errors.project}
-              helperTextInvalid={form.errors.project}
-              onChange={handleProjectUpdate}
-              required
-            />
-          )}
-        </Field>
-        {project && project.allow_override && (
+          isValid={!projectMeta.touched || !projectMeta.error}
+          helperTextInvalid={projectMeta.error}
+          onChange={handleProjectUpdate}
+          required
+        />
+        {project?.allow_override && (
           <FieldWithPrompt
             fieldId="template-scm-branch"
             label={i18n._(t`SCM Branch`)}
             promptId="template-ask-scm-branch-on-launch"
             promptName="ask_scm_branch_on_launch"
           >
-            <Field name="scm_branch">
-              {({ field }) => {
-                return (
-                  <TextInput
-                    id="template-scm-branch"
-                    {...field}
-                    onChange={(value, event) => {
-                      field.onChange(event);
-                    }}
-                  />
-                );
+            <TextInput
+              id="template-scm-branch"
+              {...scmField}
+              onChange={(value, event) => {
+                scmField.onChange(event);
               }}
-            </Field>
+            />
           </FieldWithPrompt>
         )}
-        <Field
-          name="playbook"
-          validate={required(i18n._(t`Select a value for this field`), i18n)}
-          onBlur={handleBlur}
+        <FormGroup
+          fieldId="template-playbook"
+          helperTextInvalid={playbookMeta.error}
+          isRequired
+          label={i18n._(t`Playbook`)}
         >
-          {({ field, form }) => {
-            const isValid = !form.touched.playbook || !form.errors.playbook;
-            return (
-              <FormGroup
-                fieldId="template-playbook"
-                helperTextInvalid={form.errors.playbook}
-                isRequired
-                isValid={isValid}
-                label={i18n._(t`Playbook`)}
-              >
-                <FieldTooltip
-                  content={i18n._(
-                    t`Select the playbook to be executed by this job.`
-                  )}
-                />
-                <PlaybookSelect
-                  projectId={form.values.project}
-                  isValid={isValid}
-                  form={form}
-                  field={field}
-                  onBlur={() => form.setFieldTouched('playbook')}
-                  onError={setContentError}
-                />
-              </FormGroup>
-            );
-          }}
-        </Field>
+          <FieldTooltip
+            content={i18n._(t`Select the playbook to be executed by this job.`)}
+          />
+          <PlaybookSelect
+            projectId={formikValues.project}
+            isValid={!(playbookMeta.touched || playbookMeta.error)}
+            form={formikBag}
```

Comment:
The form prop can be removed here and from PlaybookSelect...it's not actually used in AnsibleSelect